share/metainfo/org.librepcb.LibrePCB.appdata.xml

```diff
@@ -64,5 +55,9 @@
     <binary>librepcb-cli</binary>
   </provides>
 
+  <releases>
+    <release version="0.1.3" date="2019-11-30"/>
```

Comment:
Hmm since the tag is called `releases`, I wonder whether we should actually list *every* release instead of just the most recent one?

Reply:
You're right! https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-releases

I'll update.